doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.zooniverseCutouts.ZooniverseCutoutsTask.rst

```diff
@@ -0,0 +1,28 @@
+.. lsst-task-topic:: lsst.pipe.tasks.zooniverseCutouts.ZooniverseCutoutsTask
+
+#################
+ZooniverseCutoutsTask
+#################
+
+Create template/science/difference image cutouts for upload to a Zooniverse project.
```

Comment:
I've decided to remove this RST doc entirely: since we don't yet know how this will be used, we don't want to document any usage until we have that API defined.

Reply:
Then can you clarify what is the goal of DM-31046? It almost sounds like you're saying this code may need to be completely rewritten once you do know what it's for.